dist/cms.js

```diff
@@ -35,10 +35,52 @@ function getWordPressEmbedBlockCounts() {
   }
 }
 
+/**
+ * Obtains data about scripts that WordPress prints on the page.
+ *
+ * @returns {{}[]}
+ */
+function getWordPressScripts() {
+  const entries = [];
+
+  /**
+   * Checks whether the provided element is an inline script.
+   *
+   * @param {?Element} element Element to examine. May be null.
+   * @return {?number} Size of the inline script or null if the element isn't an inline script.
+   */
+  const getInlineScriptSize = (element) => {
+    if (element instanceof HTMLScriptElement && !element.src) {
+      return element.textContent.length;
+    }
+    return null;
+  };
+
+  const scripts = document.querySelectorAll('script[src][id$="-js"]');
```

Comment:
Will all scripts have an id that ends with `-js` in WordPress?

Also this will pick up scripts on other sites that conform to this standard even if they aren't WordPress sites, but think that's OK. Unless there's an easy way to detect if this is a WordPress site and limit this custom metric to just that?

Reply:
> Will all scripts have an id that ends with `-js` in WordPress?

Yes, all WordPress scripts get the `-js` suffix (cf. [source code](https://github.com/WordPress/wordpress-develop/blob/f933aa3616aa8d49bdd67886e4af287579093c25/src/wp-includes/class-wp-scripts.php#L414)).

> Also this will pick up scripts on other sites that conform to this standard even if they aren't WordPress sites, but think that's OK. Unless there's an easy way to detect if this is a WordPress site and limit this custom metric to just that?

That's true. There are various ways of sniffing for whether a site is WordPress, as can be [seen in Wappalyzer](https://github.com/wappalyzer/wappalyzer/blob/e27056248a7b9c678e5ff075558ba7b2611d434d/src/technologies/w.json#L1939-L1978). I don't suppose this `cms.js` script has access to the Wappalyzer test results? Otherwise, I'm not sure it would be worthwhile from a maintenance POV to copy the detection logic into this script.

Reply:
Yeah let's leave it as is.